timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -336,6 +335,11 @@ def save(self) -> None:
 
         self.models = models
 
+    def get_model_oof_predictions(self, model_name: str) -> TimeSeriesDataFrame:
```

Comment:
should any of this, `slice_for_scoring` and `score_with_predictions` be marked private (single underscore)?